### PR TITLE
fix: Display username when member name is missing in members list

### PR DESCRIPTION
**Pull request by Rohit3523**
## Proposed changes
Currently, when a member's name is missing, the app renders nothing, while the web client displays the username in such cases. This PR aligns the app’s behavior with the web client by ensuring the username is shown when the member name is missing. Additionally, I noticed that the divider between two members is either too light or missing, so I’ve fixed this issue by adding a `marginBottom` of *0.5* to the member component.

## Issue(s)	
none

## Screenshots
|Before|After|
| --- | --- |
| Screenshot 2024-11-13 at 12 08 54 AM | Screenshot 2024-11-13 at 12 09 37 AM |

### Web Reference
Screenshot 2024-11-13 at 12 11 31 AM

## Types of changes
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist
- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/views/RoomMembersView/index.tsx [hunk 1]
```diff
@@ -385,9 +385,9 @@ const RoomMembersView = (): React.ReactElement => {
 			<FlatList
 				data={filteredMembers || state.members}
 				renderItem={({ item }) => (
-					<View style={{ backgroundColor: colors.surfaceRoom }}>
+					<View style={{ backgroundColor: colors.surfaceRoom, marginBottom: 0.5 }}>
```
Comment (diegolmello) [inline app/views/RoomMembersView/index.tsx:388]: Can you do it on a separate PR?
This is a known issue on our app and we haven't been able to fix it.
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5879

Can you give it a try?
Thanks!

**Review by diegolmello — CHANGES_REQUESTED** [review]

File: app/views/RoomMembersView/index.tsx (hunk 1 above)
Comment (RohitBansal272) [reply app/views/RoomMembersView/index.tsx:388]: Sure 😊

Comment (Rohit3523) [reply app/views/RoomMembersView/index.tsx:388]: Sure 😊

**Comment by Rohit3523** [comment]
I removed the separator change and will make a new PR for it soon. This PR now only fixes the issue where the app shows nothing when a member’s name is missing. Now, it will display the username, matching the web client’s behavior.

**Review by diegolmello — APPROVED** [review]
